app/pages/lab/visibility.cjsx

```diff
@@ -229,6 +239,19 @@ module.exports = React.createClass
             </tbody>
           </table>
         </div>}
+
+      <br/>
```

Comment:
Maybe use css or put the paginator in a div (which should have some margin defaults) instead of using a line break tag.